targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/TARGET_EP_ATLAS/usb_stdio.cpp

```diff
@@ -16,15 +16,15 @@
  * limitations under the License.
  */
 
-#include "USBSerial.h"
-#include "platform/mbed_retarget.h"
-
 #ifndef MBED_CONF_EP_ATLAS_ENABLE_USB_STDIO_CONSOLE
 #define MBED_CONF_EP_ATLAS_ENABLE_USB_STDIO_CONSOLE 0
 #endif
 
 #if MBED_CONF_EP_ATLAS_ENABLE_USB_STDIO_CONSOLE
 
+#include "USBSerial.h"
```

Comment:
In case this `MBED_CONF_EP_ATLAS_ENABLE_USB_STDIO_CONSOLE` is enabled, this would fail to build as there is no dependency on mbed-usb.

Does this need to be part of the hal? This looks like more application functionality that anyone can do and link to mbed-usb, was this just nice to have?

Reply:
If it is enabled the application will need to include `mbed-usb` which includes `USBSerial.h`

Reply:
The documentation for `mbed_target_override_console` says `Targets may implement this to change stdin, stdout, stderr.`.

Reply:
I will add that this target is similar to the Nordic nRF52840 Dongle and therefore it does not have the usual debug UART available. USB Serial is the only way to get similar functionality to other Mbed targets.